sockeye/training.py

```diff
@@ -188,6 +296,14 @@ def create_eval_metric_composite(metric_names: List[AnyStr]) -> mx.metric.Compos
         metrics = [TrainingModel.create_eval_metric(metric_name) for metric_name in metric_names]
         return mx.metric.create(metrics)
 
+    def _set_states(self, state_names, values):
```

Comment:
doesn't this function exist in the Module implementation of MXNet already (https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/module/bucketing_module.py#L269) ?